deps/acelite/ace/Future.cpp

```diff
@@ -230,8 +230,12 @@ ACE_Future_Rep<T>::get (T &value,
       // Destructor releases the lock.
     }
 
-  value = *this->value_;
-  return 0;
+  if (value)
```

Comment:
Uhm, fixing/changing something in a dependency is something that we should not do if not properly tracked.